templates/kibana.yml.erb

```diff
@@ -1,5 +1,12 @@
 # This file is managed by Puppet, any changes will be overwritten
 
 <% scope['kibana4::config'].keys.sort.each do |key| -%>
+<% unless scope['kibana4::config'][key].is_a?(Array) -%>
```

Comment:
Use of `unless`...`else` is considered bad style [e.g](https://signalvnoise.com/posts/2699-making-sense-with-rubys-unless).